In [ ]:
import pandas as pd
data = pd.read_csv('/home/neji/Downloads/indianEco.csv', index_col=0)
print(data.head)

In [30]:
import pandas as pd
import numpy as np
data = pd.read_csv('/home/neji/Downloads/indianEco.csv', index_col=0)
avg_GDP_growth = np.round(data["GDP growth (annual %)"].mean(), decimals=2)
print(avg_GDP_growth)
avg_Inflation_rate = np.round(data["Inflation, consumer prices (annual %)"].mean(), decimals=2)
print(avg_Inflation_rate)
avg_Pop_growth = np.round(data["Population growth (annual %)"].mean(), decimals=2)
print(avg_Pop_growth)
avg_Life_expectancy = np.round(data["Life expectancy at birth, total (years)"].mean(), decimals=0)
print(avg_Life_expectancy)
avg_GDP_per_capita = np.round(data[" GDP per capita (current US$) "].mean(), decimals=2)
print(avg_GDP_per_capita)
avg_Imports = np.round(data["Imports of goods and services (% of GDP)"].mean(), decimals=2)
print(avg_Imports)
avg_Exports = np.round(data["Exports of goods and services (% of GDP)"].mean(), decimals=2)
print(avg_Exports)

4.94
7.41
1.93
57.0
575.56
12.75
10.89


## 1st Analysis
1. The Average GDP growth (annual) within the span of these 60 years was of 4.94%.
2. The Average Life expectancy (annual) within the span of these 60 years was of 57 years. 
